clang/lib/AST/ExprConstant.cpp

```diff
@@ -15314,6 +15314,32 @@ bool FloatExprEvaluator::VisitCallExpr(const CallExpr *E) {
       Result = RHS;
```

Comment:
Unrelated, but why is up here reproducing logic that's already in APFloat?

Reply:
I have no idea. I will try to fixed in future patches.
https://github.com/llvm/llvm-project/issues/111991

Reply:
I guess it was due to that the `APFloat::minnum` claims that it fellow `IEEE-754 2019 minimumNumber semantics`.
and `fmin` needs to follow the libc's behavior.
We need to fix them.